### DISCUSSION TITLE
cool thing i made !!1!

### DISCUSSION THREAD

**Discussion by Theoyeah** [DEMOLITIONDON96/Demolition-Engine#51] [Source Show & Tell]
![redcorrupted](https://user-images.githubusercontent.com/97792861/163352695-7434035a-80d8-43ba-be68-eb1fb92c7fb6.png)
PIBBY RED

**Comment by ThatOneSillyGuyDon** [comment]
Pibby is the main reason why some people hate FNF now, cause people milk it to death

**Reply by Theoyeah** [reply]
yes but altleast less than bambi

**Comment by RealDealJinx** [comment]
it took me a moment to realize its from angry birds
